### DISCUSSION TITLE
Could Leptos be used to build a Chrome Extension?

### DISCUSSION THREAD

**Discussion by philiprhoades** [leptos-rs/leptos#1869] [Q&A]
People,

Sorry about probably dumb Q . . but is this possible / sensible for an extension that needs a bit of grunt?

Thanks,

Phil.

**Comment by gbj** [comment]
Probably! I don't know anything about Chrome extensions but Leptos can do approximately anything you can do with JS in the browser, so I'd assume so.

**Comment by philiprhoades** [comment]
@gbj ,

Interesting! - I haven't made it past a "Hello World" popup for a Chrome Extension so I would probably have to pay someone to do this I think . .
